.dev/config/utils.js

```diff
@@ -0,0 +1,41 @@
+
+/**
+ * External dependencies
+ */
+const { existsSync } = require( 'fs' );
+const path = require( 'path' );
+const { realpathSync } = require( 'fs' );
+const { sync: readPkgUp } = require( 'read-pkg-up' );
```

Comment:
Sync should be reading package.json files. It looks like the main reason this method is used here is so that we can gain access to the `{ pkg, path: pkgPath }` variables It looks like we basically just see if we are using a config at that point. This is probably a bit complicated for modularity because originally core had a bit of additional functionality here that we do not need at this time.